MapboxCoreNavigation/VisualInstructionFormatter.swift

```diff
@@ -27,4 +27,56 @@ public class VisualInstructionFormatter: NSObject {
             return nil
         }
     }
+    
+    /**
+     Creates the optimal text to be displayed given a `RouteLeg` and `RouteStep` divided into primary and secondary string.
+     */
+    public func strings(leg: RouteLeg?, step: RouteStep?) -> (String?, String?) {
```

Comment:
The signature of this method is `strings(_:step:)`. Let’s think up a name that better describes the relationship between the return type (“strings”) and the first paremeter (a leg).